File: modul 5/python/PRAK505-2310817210027-MuhammadErzaRaihan.py
def Biodata(tahunLahir, namaku, asal):
    tahun_sekarang = 2023
    umur = tahun_sekarang - tahunLahir

    print("Perkenalkan Nama Saya:", namaku)
    print("Umur Saya:", umur)
    print("Saya Adalah Angkatan:", tahun_sekarang)
    print("Asal Saya dari: ", asal)

tahunLahir = int(input())
namaku = input()
asal = input()

Biodata(tahunLahir, namaku, asal)

File: modul 5/c/PRAK505-2310817210027-MuhammadErzaRaihan.c
#include <stdio.h>

void Biodata(int tahunLahir, char *namaku, char *asal){
    int tahun_sekarang = 2023;
    int umur = tahun_sekarang - tahunLahir;

    printf("Perkenalkan Nama Saya: %s\n", namaku);
    printf("Umur Saya: %d \n", umur);
    printf("Saya Adalah Angkatan: %d\n", tahun_sekarang);
    printf("Asal Saya dari: %s\n", asal);
    
    
    
}

int main() {
    int tahunLahir;
    char namaku[20], asal[15];
    
    scanf(" %d",&tahunLahir);
    scanf(" %[^\n]%*c",&namaku);
    scanf(" %[^\n]%*c",&asal);

    Biodata(tahunLahir, namaku, asal);
    return 0;
}